app/src/main/java/io/intrepid/contest/screens/sendinvitations/selectcontacts/SelectContactsViewHolder.java

```diff
@@ -51,11 +54,19 @@ public void bindData(Contact contact) {
             emailTextView.setVisibility(GONE);
         }
 
-        byte[] photo = contact.getPhoto();
-        if (photo == null) {
-            photoImageView.setImageResource(R.drawable.default_user_photo);
+        if (contact.isSelected()) {
+            photoImageView.setImageResource(R.drawable.selected_contact_icon);
         } else {
-            photoImageView.setImageBitmap(BitmapFactory.decodeByteArray(photo, 0, photo.length));
+            byte[] photo = contact.getPhoto();
+            if (photo == null) {
+                photoImageView.setImageResource(R.drawable.default_user_photo);
+            } else {
+                photoImageView.setImageBitmap(BitmapFactory.decodeByteArray(photo, 0, photo.length));
+            }
         }
     }
+
+    public interface ContactClickListener {
+        void onContactClick(View itemView);
```

Comment:
I'm always afraid of giving any responsibility to views, but now I understand better that it was just handling its own data.
I refactored it, and it really simplified my code! Thanks for the feedback!

Can you please take a last look to make sure I covered it all? @abassawo @kentonguan